packages/vue3/src/components/Modal/Modal.vue

```diff
@@ -45,15 +52,20 @@ onUnmounted(() => {
         <slot />
       </div>
       <div class="footer">
-        <PrimaryButton v-if="!cancelOnly" @click="emit('onConfirm')">
-          <template v-if="confirmIcon" #icon>
-            <i :class="confirmIcon" />
-          </template>
-          <span>{{ confirmLabel }}</span>
-        </PrimaryButton>
-        <SecondaryButton @click="close">
-          <span>{{ cancelLabel }}</span>
-        </SecondaryButton>
+        <div v-if="!!slots.footer">
```

Comment:
Since we'll be using $slots, we can do the check like so:
```suggestion
        <div v-if="$slots.footer">
```

Reply:
FIxed [4423478](https://github.com/youcan-shop/youcan-ui/pull/427/commits/4423478828714e8edcf968d7d5a5f2baf9c0c94a)